api/go.mod

```diff
@@ -22,3 +22,27 @@ require (
 	google.golang.org/grpc v1.46.0
 	gopkg.in/yaml.v2 v2.4.0
 )
+
+require (
+	github.com/beevik/etree v1.1.0 // indirect
```

Comment:
How did we get all these new dependencies?

Reply:
`go mod tidy` added them.